docs/mkdocs.yml

```diff
@@ -72,27 +73,58 @@ extra_javascript:
 nav:
   - InvenTree:
     - InvenTree: index.md
-    - Features: features.md
     - FAQ: faq.md
+    - Demo: demo.md
+    - Release Notes: releases/release_notes.md
     - Core Concepts:
       - Terminology: concepts/terminology.md
       - Physical Units: concepts/units.md
+      - Companies: concepts/company.md
       - Custom States: concepts/custom_states.md
       - Pricing: concepts/pricing.md
-      - Threat Model: concepts/threat_model.md
+      - Project Codes: concepts/project_codes.md
+      - Barcodes:
+        - Barcode Support: barcodes/index.md
+        - Internal Barcodes: barcodes/internal.md
+        - External Barcodes: barcodes/external.md
+        - Custom Barcodes: barcodes/custom.md
     - Development:
       - Contributing: develop/contributing.md
       - Devcontainer: develop/devcontainer.md
       - React Frontend: develop/react-frontend.md
-    - Privacy: privacy.md
-    - Release Notes: releases/release_notes.md
-    - Project:
+    - Mobile App:
+      - Mobile App: app/index.md
+      - Connect: app/connect.md
+      - Navigation: app/navigation.md
+      - Search: app/search.md
+      - Barcodes: app/barcode.md
+      - Parts: app/part.md
+      - Stock: app/stock.md
+      - Purchase Orders: app/po.md
+      - Sales Orders: app/so.md
+      - Settings: app/settings.md
+      - Privacy: app/privacy.md
+      - Translation: app/translation.md
+      - Suggestions: app/issues.md
+    - InvenTree API:
+      - Overview: api/index.md
+      - API Schema: api/schema.md
+      - Model Metadata: api/metadata.md
+      - Download Data: api/download.md
+      - Bulk Delete: api/bulk_delete.md
+      - Interactive API: api/browse.md
+    - Python Interface:
+      - Overview: api/python/index.md
+      - Currency Support: api/python/currency.md
+      - Examples: api/python/examples.md
+    - Project Details:
       - Governance: project/governance.md
       - Project Security: project/security.md
       - Resources: project/resources.md
-    - Security: security.md
+      - Privacy: project/privacy.md
+      - Threat Model: project/threat_model.md
```

Comment:
The Thread Model regards individual instances, not the project; not sure if this reordering makes sense

Reply:
Oh good point. I'll fix that.